models/LogLevel.js

```diff
@@ -1,11 +1,10 @@
 module.exports = require('./Enum').create({
   values: [
-    'FATAL',
-    'ERROR',
-    'WARN',
-    'INFO',
-    'DEBUG',
-    'TRACE'
-  ],
-  autoLowerCase: true
```

Comment:
I am fine with `logLevel: 'error'` in the config or using lowercase in the environment variables, but ultimately, there should be a consistency in how they're referenced under the hood or directly referenced in the code. So `autoUpperCase` or `autoLowerCase` should be in place.